doc/md/guides/clickhouse/getting-started.mdx

```diff
@@ -0,0 +1,536 @@
+---
+id: getting-started-clickhouse
+title: Manage ClickHouse with Atlas
+slug: /guides/clickhouse
+tags: [clickhouse]
+---
+import InstallationInstructions from '../../components/_installation_instructions.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+[ClickHouse](https://clickhouse.com/), one of the prominent columnar databases, is designed for real-time analytics,
+providing exceptional speed and efficiency in handling large datasets.
+
+Many teams adopt ClickHouse for its speed and cost-efficiency only to discover that the day 2 operations of managing its
+schema can be a puzzle. Sure, if your schema contains a handful of tables that rarely change,
+you’re probably not going to feel much of this pain. But for mission-critical applications, managing complex and interconnected
+schemas while moving fast without breaking things becomes difficult.
+
+#### Enter: Atlas
+Atlas helps developers manage their database schema as code - abstracting away the intricacies of database schema
+management. With Atlas, users provide the desired state of the database schema and Atlas automatically plans the
+required migrations.
+
+In this guide, we will dive into setting up Atlas for ClickHouse, and introduce the different workflows available.
+
+## Prerequisites
+1. A ClickHouse database
+2. Docker
+3. Atlas installed on your machine:
+<InstallationInstructions />
+
+## Logging in to Atlas
+
+To use ClickHouse with Atlas, you'll need to [log in to Atlas](https://auth.atlasgo.cloud/signup). If it's your first time,
+you will be prompted to create both an account and a workspace (organization):
+
+<Tabs>
+<TabItem value="web" label="Via Web">
+
+```shell
+atlas login
+```
+
+</TabItem>
+<TabItem value="token" label="Via Token">
+
+```shell
+atlas login --token "ATLAS_TOKEN"
+```
+
+</TabItem>
+<TabItem value="env-var" label="Via Environment Variable">
+
+```shell
+ATLAS_TOKEN="ATLAS_TOKEN" atlas login
+```
+
+</TabItem>
+</Tabs>
+
+## Inspecting our Database
+
+Let's start off with a minimal database with a `users` table and an `id` as the primary key.
+
+```sql
```

Comment:
How is the initial database created? 

How is this schema created in clickhouse?